modules/products/bitbucket/variables.tf

```diff
@@ -172,4 +172,14 @@ variable "db_master_password" {
   description = "Master password for the RDS instance."
   type        = string
   default     = null
+}
+
+variable "shared_home_snapshot_id" {
+  description = "EBS Snapshot ID with shared home content."
+  type        = string
+  default     = null
+  validation {
+    condition     = var.shared_home_snapshot_id == null || can(regex("^.{1,255}$", var.shared_home_snapshot_id))
```

Comment:
```suggestion
    condition     = var.shared_home_snapshot_id == null || can(regex("^snap-\\w{17}$", var.shared_home_snapshot_id))
```

Reply:
Is this suggestion because of the snapshot name created in AWS? Can user customise the snapshot name, for instance if create it by CLI or any other tool?

Reply:
No, it is always this snapshot ID, there is no other accepted format in the terraform module. I have based this regex on the snapshots across regions and old public ones. Seems that the format didn't change and it is always similar to: `snap-0c00767b17c3cca7f`